### PR TITLE
Example requests for VA Forms and Appeals APIs

### PR DESCRIPTION
**Pull request by caseywilliams**
## Summary

- This replaces our current Insomnia request examples with configs for [Bruno](https://www.usebruno.com/), which is much better in terms of version control of configs.
- [Docs for Bruno here](https://docs.usebruno.com/)
- **[Link to separate PR with instructions on how to use these configs](https://github.com/department-of-veterans-affairs/banana-peels/pull/25)**

## Related issue(s)
None

## Testing done

- Ran all example requests in Bruno

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
None

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

File: modules/appeals_api/.bruno_appeals_api/helpers.js
Comment (caseywilliams) [inline modules/appeals_api/.bruno_appeals_api/helpers.js:1]: This JS file replicates the script provided by the Pivot team for Postman, [which is here](https://github.com/department-of-veterans-affairs/vets-api-clients/tree/master/samples/oauth_ccg_postman_pre_request_script). I adjusted it a little bit to work with the built-in libraries that Bruno ships (includes uuid and axios), and to manage the `jsrsasign` dependency using `npm` instead of an ad-hoc download script. I'll include instructions about this in the docs for these example requests.

File: modules/appeals_api/.bruno_appeals_api/higher-level-reviews v0/Create HLR.bru
Comment (caseywilliams) [inline modules/appeals_api/.bruno_appeals_api/higher-level-reviews v0/Create HLR.bru:143]: Here's an example of using the oauth-token-fetching helper I added. It uses the `vars:pre-request` setting for `oauth_scope` that's configured just above this line, and fetches a token with that scope.

**Comment by caseywilliams** [comment]
I've added two extra commits here:
- The first adds `.bru` files to the ignorelist for in the `Dangerfile` so that they are not counted in the number of lines changed
- The second corrects the expansion of a bash variable in the CODEOWNERS check script - it was previously looping over the contents of this variable in a way that incorrectly categorized filenames with spaces in them (you can see the comments above complaining about "Please update the .github/CODEOWNERS file and add the entry for the Offending file: v0/Docs.bru")

**Comment by caseywilliams** [comment]
I added one more commit to fix the script that does the "Audit Service Tags" PR check - it wasn't able to handle filenames with spaces in them before, either.

**Comment by mathisto** [comment]
> I've added two extra commits here:
> 
> * The first adds `.bru` files to the ignorelist in the `Dangerfile` so that they are not counted in the number of lines changed
> * The second corrects the expansion of a bash variable in the CODEOWNERS check script - it was previously looping over the contents of this variable in a way that incorrectly categorized filenames with spaces in them (you can see the comments above complaining about `Please update the .github/CODEOWNERS file and add the entry for the Offending file: v0/Docs.bru` - this was because of the lack of parameter expansion)

Wow, great catch!

**Review by mathisto — DISMISSED** [review]

**Review by mathisto — APPROVED** [review]

File: .gitignore
Comment (kristen-brown) [inline .gitignore:99]: @caseywilliams Did you intend to commit some of these git files?

**Review by kristen-brown — CHANGES_REQUESTED** [review]

**Review by kristen-brown — APPROVED** [review]

**Comment by kristen-brown** [comment]
This looks like one that the Platform should review/approve before merging, given the CODEOWNERS script changes.

File: .gitignore
Comment (rmtolmach) [inline .gitignore:99]: When do you encounter `node_modules` files?

Comment (caseywilliams) [reply .gitignore:99]: `node_modules` is a folder created automatically by node/javascript package managers like `npm` or `yarn` whenever you install dependencies (similar to how `bundle install` downloads all the files for each bundled gem). It's used to hold all the dependency packages listed in a javascript project's `package.json` file (which is similar to a `Gemfile` in ruby). The `package.json` that I added in this PR includes a single dependency, which is used for signing JWTs with an RSA key. These `node_modules` files are not committed to version control because they are just copies of the dependencies that can be automatically downloaded by anyone via a package manager.

Comment (rmtolmach) [reply .gitignore:99]: ok, sounds like I probably won't encounter a node_modules directory myself, but it makes sense to add it here.

**Review by rmtolmach — APPROVED** [review]

**Pull request merged**
